modules/webbrowser/src/renderhandlergl.cpp

```diff
@@ -44,7 +44,41 @@ bool RenderHandlerGL::GetViewRect(CefRefPtr<CefBrowser> browser, CefRect& rect)
     return true;
 }
 
-void RenderHandlerGL::OnPaint(CefRefPtr<CefBrowser> browser, PaintElementType,
+void RenderHandlerGL::OnPopupShow(CefRefPtr<CefBrowser> browser, bool show) {
+    if (!show) {
+        // Clear the popup rectangle.
+        ClearPopupRects();
+    }
+}
+
+void RenderHandlerGL::OnPopupSize(CefRefPtr<CefBrowser> browser, const CefRect& rect) {
```

Comment:
Would it be possible to use this for tooltips in glfw?

Reply:
Yes, it should :)